less/v2/apps.less

```diff
@@ -1314,6 +1314,30 @@ dialog.dnd5e2.application {
   }
 }
 
+/* ---------------------------------- */
+
+.dnd5e2.damages-config.trait-selector {
+  .traits {
+    grid-template-columns: [icon] 16px [name] 2fr [proficiency] 1fr;
+    .trait-list li {
+      dnd5e-icon, i {
+        --icon-fill: var(--color-text-light-6);
+        color: var(--color-text-light-6);
+      }
+
+      > i {
+        font-family: var(--font-awesome);
+        font-weight: bold;
+        font-style: normal;
```

Comment:
I'm wondering if we want to center these to create a bit more balance:

<img width="289" alt="Screenshot 2024-10-23 at 14 31 52" src="https://github.com/user-attachments/assets/e06f3f4f-bd8c-49c0-a59a-bfe6f2ac3c4a">
<img width="293" alt="Screenshot 2024-10-23 at 14 31 41" src="https://github.com/user-attachments/assets/6761a969-8a95-4d2a-82b1-4cdf29290112">